index.bs

```diff
@@ -350,59 +355,51 @@ interface VRPose {
 };
 </pre>
 
-### Attributes ### {#vrpose-state-attributes}
-
-<dfn attribute for="VRPose">timestamp</dfn>
-Monotonically increasing value that allows the author to determine if position
-state data been updated from the hardware. Since values are monotonically
-increasing, they can be compared to determine the ordering of updates, as newer
-values will always be greater than or equal to older values. The timestamp starts at 0 the first time getPose is invoked for a given display.
+### Attributes ### {#vrpose-attributes}
 
 <dfn attribute for="VRPose">position</dfn>
-Position of the {{VRDisplay}} at {{timestamp}} as a 3D vector. Position is given
-in meters from an origin point, which is either the position the sensor was
-first read at or the position of the sensor at the point that {{resetPose()}}
-was last called. The coordinate system uses these axis definitions:
+Position of the {{VRDisplay}} as a 3D vector. Position is given in meters from
+an origin point, which is either the position the sensor was first read at or
+the position of the sensor at the point that {{resetPose()}} was last called.
+The coordinate system uses these axis definitions:
 
 * Positive X is to the user's right.
 * Positive Y is up.
 * Positive Z is behind the user.
 
 All positions are given relative to the identity orientation in sitting space.
-Transforming this point with {{VRStageParameters}}.{{sittingToStandingTransform}}</code>
-converts this to standing space. MAY be NULL if the sensor is incapable of
-providing positional data. User agents MAY provide emulated position values
-through techniques such as neck modeling, but when doing so SHOULD report
-{{VRDisplayCapabilities}}.{{hasPosition}} as false. When not NULL MUST be a
-three-element array.
+MAY be NULL if the sensor is incapable of providing positional data. User agents
+MAY provide emulated position values through techniques such as neck modeling,
+but when doing so SHOULD report {{VRDisplayCapabilities}}.{{hasPosition}} as
+false. When not NULL MUST be a three-element array.
 
 <dfn attribute for="VRPose">linearVelocity</dfn>
-Linear velocity of the sensor at {{timestamp}} meters per second. MAY be NULL if
-the sensor is incapable of providing linear velocity. When not NULL MUST be a
+Linear velocity of the sensor given in meters per second. MAY be NULL if the
+sensor is incapable of providing linear velocity. When not NULL MUST be a
 three-element array.
 
 <dfn attribute for="VRPose">linearAcceleration</dfn>
-Linear acceleration of the sensor at {{timestamp}} given in meters per second
-squared. MAY be NULL if the sensor is incapable of providing linear
-acceleration. When not NULL MUST be a three-element array.
+Linear acceleration of the sensor given in meters per second squared. MAY be
+NULL if the sensor is incapable of providing linear acceleration. When not NULL
```

Comment:
I'm a C++ developer, damnit! (Oh, wait.... that's `nullptr` now. Doh!)

Fixed, will push soon.